### PR TITLE
Feature/892 coinstac api server test

### LINKED ISSUE #892: Coinstac api server test

### PR DESCRIPTION
**Pull request by kalitopix**
# Problem
referenced issue(s):

# Solution

# Testing

closes #892

### CODE REVIEW

**Review by rochaeb — APPROVED** [review]

**Comment by rssk** [comment]
Needs some merge conf fixes, note the newer test-setup

**Review by rssk — CHANGES_REQUESTED** [review]
re: merge conf

File: packages/coinstac-api-server/tests/resolvers.test.js [hunk 1]
```diff
@@ -0,0 +1,1027 @@
...
+
+  await helperFunctions.validateUniqueUser(req2, (error) => {
+    t.is(error.output.payload.message, 'Email taken');
+  });
+
+  const req3 = { payload: { username: 'newuser', email: 'newuser@mrn.org' } };
+  const res = sinon.spy();
+
+  await helperFunctions.validateUniqueUser(req3, res);
+  t.true(res.called);
```
Comment (rssk) [inline packages/coinstac-api-server/tests/resolvers.test.js]: this is just saying this callback was called right? Could it have been called with an error? Why not just check if the error obj is false?

File: packages/coinstac-api-server/tests/resolvers.test.js [hunk 2]
```diff
@@ -0,0 +1,1027 @@
...
+
+  axios.post.restore();
+
+  sinon.stub(axios, 'post').rejects(new Error('error'));
+
+  const error2 = await Mutation.createRun(auth, { consortiumId: consortiumId2 });
+  t.is(error2.output.payload.statusCode, 406);
+
+  axios.post.restore();
+});
```
Comment (rssk) [inline packages/coinstac-api-server/tests/resolvers.test.js:506]: nice hitting all the different states here 👍

File: packages/coinstac-api-server/tests/resolvers.test.js [hunk 3]
```diff
@@ -0,0 +1,1027 @@
...
+
+  const res = await Mutation.updatePassword(auth, args);
+  t.is(getMessageFromError(res), INVALID_PASSWORD);
+
+  args = {
+    currentPassword: 'password',
+    newPassword: 'admin',
+  };
+
+  await Mutation.updatePassword(auth, args);
```
Comment (rssk) [inline packages/coinstac-api-server/tests/resolvers.test.js]: should there be a test after this? Why not check if the password was updated via validate?

File: packages/coinstac-api-server/tests/resolvers.test.js [hunk 4]
```diff
@@ -0,0 +1,1027 @@
...
+//     mutation deleteConsortiumById($consortiumId: ID) {
+//       deleteConsortiumById(consortiumId: $consortiumId) {
+//         id
+//         name
+//       }
+//     }
+//   `, auth, null, { consortiumId: consortium.id.toString() });
+
+//   subDataControl.unsubscribe();
+// });
```
Comment (rssk) [inline packages/coinstac-api-server/tests/resolvers.test.js]: ?

**Review by rssk — CHANGES_REQUESTED** [review]
Looking good, just a few comments and changes

**Review by rssk — APPROVED** [review]

**Pull request merged**